TombEngine/Scripting/Internal/TEN/Misc/Miscellaneous.cpp

```diff
@@ -142,55 +142,34 @@ namespace Misc
 		return Camera.oldType;
 	}
 	
-	/// Play an audio track
-	//@function PlayAudioTrack
-	//@tparam string name of track (without file extension) to play
-	//@tparam Misc.SoundTrackType type of the audio track to play
 	static void PlayAudioTrack(const std::string& trackName, TypeOrNil<SoundTrackType> mode)
```

Comment:
As I said earlier on Discord, I think that deprecated variations should be removed, if they cause warnings in console.

Reply:
Fixed References to Sound.SoundTrackType